src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1644,178 +1644,175 @@ public static void markErrored() {
-			if (pluginPackages.isEmpty()) {
-				logEx("You should report it at " + issuesUrl + ". Please copy paste this report there (or use paste service).");
-				logEx("This ensures that your issue is noticed and will be fixed as soon as possible.");
-			} else {
-				logEx("It looks like you are using some plugin(s) that alter how Skript works (addons).");
-				if (stackPlugins.isEmpty()) {
-					logEx("Here is full list of them:");
-					StringBuilder pluginsMessage = new StringBuilder();
-					for (PluginDescriptionFile desc : pluginPackages.values()) {
-						pluginsMessage.append(desc.getFullName());
-						String website = desc.getWebsite();
-						if (website != null && !website.isEmpty()) // Add website if found
-							pluginsMessage.append(" (").append(desc.getWebsite()).append(")");
-
-						pluginsMessage.append(" ");
-					}
-					logEx(pluginsMessage.toString());
-					logEx("We could not identify which of those are specially related, so this might also be Skript issue.");
-				} else {
-					logEx("Following plugins are probably related to this error in some way:");
-					StringBuilder pluginsMessage = new StringBuilder();
-					for (PluginDescriptionFile desc : stackPlugins) {
-						pluginsMessage.append(desc.getName());
-						String website = desc.getWebsite();
-						if (website != null && !website.isEmpty()) // Add website if found
-							pluginsMessage.append(" (").append(desc.getWebsite()).append(")");
-
-						pluginsMessage.append(" ");
-					}
-					logEx(pluginsMessage.toString());
-				}
+			logEx("An unexpected error occurred with Skript. This issue is likely not your fault.");
+			logExAddonInfo(issuesUrl, stackPlugins);
+		}
+	}
 
-				logEx("You should try disabling those plugins one by one, trying to find which one causes it.");
-				logEx("If the error doesn't disappear even after disabling all listed plugins, it is probably Skript issue.");
-				logEx("In that case, you will be given instruction on how should you report it.");
-				logEx("On the other hand, if the error disappears when disabling some plugin, report it to author of that plugin.");
-				logEx("Only if the author tells you to do so, report it to Skript's issue tracker.");
+	private static void logExAddonInfo(String issuesUrl, Set<PluginDescriptionFile> stackPlugins) {
+		if (pluginPackages.isEmpty()) {
+			logEx("Report the issue: " + issuesUrl);
+		} else {
+			logEx("You are using some plugins that alter how Skript works (addons).");
+			if (stackPlugins.isEmpty()) {
+				logEx("Full list of addons:");
+				pluginPackages.values().forEach(desc -> logEx(getPluginDescription(desc)));
+				logEx("We could not identify related addons, it might also be a Skript issue.");
+			} else {
+				logEx("The following plugins are likely related to this error:");
+				stackPlugins.forEach(desc -> logEx(getPluginDescription(desc)));
 			}
+			logEx("Try disabling the listed plugins one by one to identify the cause.");
```

Comment:
this should include that if disabling a plugin causes the error to be fixed, it should be reported to the plugin developer to reduce the chances of invalid bug issues

Reply:
I think we should also probably clarify disabling a plugin. I dont think we want people to use plugman or something.

Reply:
maybe change disabling to remove?

Reply:
maybe "temporarily removing"? i'm cool with either